src/main/java/dog/pawbook/model/owner/Owner.java

```diff
@@ -16,10 +16,11 @@
 public class Owner {
 
     // Identity fields
+    private static int userIDCounter = 0;
```

Comment:
this might need to be a part of the addressbook instead, given that storage will likely modify it after reading back the saved file, and especially since in the tests there will be different addressbooks created, but a simple equal will fail since there is no way to sync up the id numbers

Reply:
I agree, I think the userIDcounter should be implemented perhaps in addressbook instead as certain tests fail currently due to ID not syncing up